futures-util/src/future/always_ready.rs

```diff
@@ -0,0 +1,46 @@
+use super::assert_future;
+use core::pin::Pin;
+use futures_core::future::{FusedFuture, Future};
+use futures_core::task::{Context, Poll};
+
+/// Future for the [`always_ready`](always_ready()) function.
+#[derive(Debug, Clone, Copy)]
```

Comment:
derive creates trait impl that have trait bounds for all generics. In this case, they are `T: Copy, F: Copy` for `impl Copy for AlwaysReady`, `T: Debug, F: Debug` for `impl Debug for AlwaysReady`, but `T: Copy` bound and all `Debug` bounds are useless.

Could you replace these derives with manual impls?

Reply:
Done, I required `Debug` for `F`, but if you give the desired debug output I can relax that bound aswell.

Reply:
> I required `Debug` for `F`, but if you give the desired debug output I can relax that bound aswell.

Other our futures/streams don't print closure (https://github.com/rust-lang/futures-rs/pull/1677), so I would like to adopt the same behavior (like [PollFn](https://github.com/rust-lang/futures-rs/pull/1677/files#diff-72ac786dc163d40e11613729e088985b6a3def2c2182270c11303b77273be0b1R45) does) here.

Reply:
Okay, done